src/cdk/tree/tree.ts

```diff
@@ -279,26 +279,13 @@ export class CdkTree<T> implements AfterContentChecked, CollectionViewer, OnDest
   }
 }
 
-
-/**
- * Tree node for CdkTree. It contains the data in the tree node.
- */
-@Directive({
-  selector: 'cdk-tree-node',
-  exportAs: 'cdkTreeNode',
-  host: {
-    '[attr.aria-expanded]': 'isExpanded',
-    '[attr.aria-level]': 'role === "treeitem" ? level : null',
-    '[attr.role]': 'role',
-    'class': 'cdk-tree-node',
-  },
-})
-export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
+/** Base class with the functionality of a `CdkTreeNode`. */
+export class CdkTreeNodeBase<T> implements FocusableOption, OnDestroy {
```

Comment:
I would've liked to avoid having to create this class, however having the `CdkNestedTreeNodeBase` extend from `CdkTreeNode` causes the compiler to throw an error.